lib/storage/partition.go

```diff
@@ -1316,7 +1316,7 @@ func (pt *partition) runFinalDedup() error {
 	t := time.Now()
 	logger.Infof("starting final dedup for partition %s using requiredDedupInterval=%d ms, since the partition has smaller actualDedupInterval=%d ms",
 		pt.bigPartsPath, requiredDedupInterval, actualDedupInterval)
-	if err := pt.ForceMergeAllParts(); err != nil {
+	if err := pt.ForceMergeAllParts(pt.stopCh); err != nil {
```

Comment:
would it make sense to use the table channel here? 
From my understandig, this should improve shutdown time if vmstorage is in the middle of the final dedup phase.
With the current approach, vmstorage will have to wait for finalDedup to finish. wdyt?